src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -55,6 +55,12 @@ def load_arguments(self, _):
     no_prompt = CLIArgumentType(arg_type=get_three_state_flag(), help='The option to disable the prompt of missing parameters for ARM template. '
                                 'When the value is true, the prompt requiring users to provide missing parameter will be ignored. The default value is false.')
 
+    deployment_what_if_result_format_type = get_enum_type(WhatIfResultFormat, WhatIfResultFormat.full_resource_payloads)
+    deployment_what_if_no_pretty_print_type = CLIArgumentType(options_list=['--no-pretty-print'], action='store_true',
+                                                              help='Disables pretty-print for What-If results. When set, the output format type will be used.')
+    deployment_what_if_confirmation_type = CLIArgumentType(options_list=['--confirm-with-what-if'], action='store_true',
+                                                           help='Instructs the command to run deployment What-If before excuting the deployment. It then prompts you to acknowledge resource changes before it continues.')
```

Comment:
Please use active voice, like `Instructs the command` -> `Instruct the command`

Reply:
Changed.